src/gui/MainWindow.cpp

```diff
@@ -552,13 +555,20 @@ void MainWindow::finalize()
 								m_toolBar );
 	controllers_window->setShortcut( Qt::Key_F11 );
 
+    m_countDownClock = new LcdSpinBox(2, this, tr("Count Down Clock"));
+
+    m_countDownClock->setLabel( tr("SEC") );
```

Comment:
In the final version don't use 1250 like a magic number, cause the TicksPerTact can change in the future. It's better to have a constant that's `24000/TicksPerTact`